scientist-softserv/palni-palci
# 530 - Bring Back Based Near

# Story
Fixes/Re-Enables the `based_near` field so that an error is not thrown upon trying to create a work with based near location set

TODO: 
- update the label for this field to location?

# Related
- #530

# Expected Behavior Before Changes
Based near was turned off

# Expected Behavior After Changes
- users can select based near geography location for their works
- users can select multiple locations

# Screenshots / Video
Video
https://share.getcloudapp.com/P8uXkxmg

<details>
<summary>Form field with finished help text (since it was not finished in the video)</summary>
<img width="925" alt="image" src="https://github.com/scientist-softserv/palni-palci/assets/73361970/1354500b-afff-418d-8533-7a669789797a">

</details>

# Notes/Testing Instructions
In order for this field to work, you will need to configure the geonames in each tenant. Follow the following steps to do this: 

### Steps to Enable Based Near Controlled Vocabulary in Hyku/Hyrax app  
1. Create a free account on [geonames.org](http://geonames.org/)
2. Go to your account settings at https://www.geonames.org/manageaccount and select the option at the bottom to enable free services. 
3. Go into your hyrax/hyku account settings in the dashboard
4. Enter your geonames username in the hyrax account settings
5. You should now be able to see the list of locations in the based near field inside the forms

Then, to test
1. create a work with the location field filled in, make sure it saves.
2. edit that work and make sure it saves
